jackal_description/urdf/accessories/pointgrey_bumblebee2_mount.urdf.xacro

```diff
@@ -0,0 +1,53 @@
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find pointgrey_camera_description)/urdf/pointgrey_bumblebee2.urdf.xacro" />
+
+  <xacro:macro name="pointgrey_bumblebee2_mount" params="prefix name tilt">
+
+    <BB2-08S2C-38 frame="${prefix}_camera" name="${name}" />
+
+    <link name="${prefix}_camera_mount">
+      <visual>
+        <origin xyz="0 0 0" rpy="1.5707 0 1.5707" />
+        <geometry>
+          <!-- Origin of this mesh is the base of the mount. -->
+          <mesh filename="package://jackal_description/meshes/pointgrey-camera-bracket.stl" scale="0.001 0.001 0.001" />
```

Comment:
It's meant to be a generic bracket supporting any ISO 1222:2010 compliant camera.